vllm/worker/habana_model_runner.py

```diff
@@ -1850,8 +2044,10 @@ def _check_config(self, batch_size, seq_len, is_prompt, warmup_mode):
         self.seen_configs.add(cfg)
         if not seen and not warmup_mode:
             phase = 'prompt' if is_prompt else 'decode'
-            logger.warning("Configuration: (%s, %s, %s) was not warmed-up!",
-                           phase, batch_size, seq_len)
+            if not self.calibrate_buckets:
+                logger.warning(
```

Comment:
Do we need to divide code into 3 lines? We now have wide displays and it does nt make it more readable

Reply:
> Do we need to divide code into 3 lines? 

I wish we didn't, but format.sh made this into such an abomination.